OpenRA.Mods.Common/Projectiles/InstantHit.cs

```diff
@@ -71,7 +71,11 @@ public void Tick(World world)
 
 			if (!doneDamage)
 			{
-				args.Weapon.Impact(Target.FromPos(target), args.SourceActor, args.DamageModifiers);
+				if (info.Inaccuracy.Length > 0)
```

Comment:
I meant that `target` is currently a WPos, and blockability and inaccuracy are designed around that. And the way I understand your comment, I'm supposed to change `target` to Target instead of WPos?